en/app.ftl

```diff
@@ -460,11 +462,36 @@ onboarding-premium-domain-description = With a custom domain, you can make your
 onboarding-premium-reply-title = Reply to emails without giving away your real address
 onboarding-premium-reply-description = Need to reply to emails sent to an alias? Just reply as normal — your alias will still protect your email address.
 
+## Multi-part Premium Onboarding - This is a walk-through onboarding expereince
```

Comment:
Add a line space for section comment

Reply:
Added in 9731f8f389e67df60bf7cdfb43fa172f4f896355